cobigen-angular-client-generation/index.asciidoc

```diff
@@ -0,0 +1,30 @@
+= CobiGen Angular client generation
+====
+With the CobiGen code generator you can easily generate components for your angular application.
+The following tutorial provides an example for using the CobiGen cli.
+You can find more information on https://github.com/devonfw/cobigen/wiki/howto_angular-client-generation
+====
+
+[step]
+--
+restoreDevonfwIde(["java", "mvn", "node", "ng", "npm"])
+--
+
+For this tutorial an installation of the devonfw IDE is required, which was already set up for you. The next step is to install the CobiGen CLI.
+[step]
+--
+installCobigen()
```

Comment:
I think the pipeline fails because you wrote `installCobigen` instead of `installCobiGen`. The upper and lower case letters matter